site/docs/configuration.md

```diff
@@ -109,14 +110,14 @@ spark.read
     .table("catalog.db.table")
 ```
 
-| Spark option    | Default               | Description                                                                               |
-| --------------- | --------------------- | ----------------------------------------------------------------------------------------- |
-| snapshot-id     | (latest)              | Snapshot ID of the table snapshot to read                                                 |
-| as-of-timestamp | (latest)              | A timestamp in milliseconds; the snapshot used will be the snapshot current at this time. |
-| split-size      | As per table property | Overrides this table's read.split.target-size and read.split.metadata-target-size         |
-| lookback        | As per table property | Overrides this table's read.split.planning-lookback                                       |
-| file-open-cost  | As per table property | Overrides this table's read.split.open-file-cost                                          |
-
+| Spark option               | Default               | Description                                                                               |
+| -------------------------- | --------------------- | ----------------------------------------------------------------------------------------- |
+| snapshot-id                | (latest)              | Snapshot ID of the table snapshot to read                                                 |
+| as-of-timestamp            | (latest)              | A timestamp in milliseconds; the snapshot used will be the snapshot current at this time. |
+| split-size                 | As per table property | Overrides this table's read.split.target-size and read.split.metadata-target-size         |
+| lookback                   | As per table property | Overrides this table's read.split.planning-lookback                                       |
+| file-open-cost             | As per table property | Overrides this table's read.split.open-file-cost                                          |
+| use-approximate-statistics | As per table property | Overrides this table's read.spark.read.spark.use-approximate-statistics                   |
```

Comment:
I don't think we need a table option for this. If we were going to return incorrect stats, then I would want a flag to enable or disable it. But because we are going to use table-level stats, we can detect when to do it based on whether or not there are filters. No filter, then use table level stats.

Reply:
what if manifest size is still large after filters are applied?  user may want to disable scanning entire manifest if not trying to broadcast table

Reply:
I'm not sure I understand the case you're talking about. What I'm saying is that because we are using reliable stats that are maintained in the table metadata, we can always use them if there are no filters. That takes care of the bad case in Spark 2.4. Since Spark 2.4 doesn't ever push filters before calling `estimateStatistics`, it will always use metadata stats and will avoid the issue entirely.

Reply:
thanks for reply. to clarify I usecase is while using spark3 we still want to disable estimating statistics by reading manifest ( assuming client code is reading large portion of the table)

Reply:
we can discuss this later , I will cleanup PR to remove parameter and always look at filter to decide `estimateStatistics` behaviour. thanks